SwiftI18n/Classes/Case/I18nCaseI18nCaseTransform.swift

```diff
@@ -22,6 +23,8 @@ public enum I18nCaseTransform: String {
             return string.lowercased()
         case .capitalized:
             return string.capitalized(with: Locale(identifier: I18nManager.instance.language))
+        case .firstCapitalized:
+            return String(string.uppercased().prefix(1) + string.lowercased().dropFirst())
```

Comment:
TBH I don't think this is a good approach.
Languages are hard and have a lot of different rules for different languages.
Changes like this should not be done in code. They should be done in translations, by translators.

Reply:
I agree with you but you know we cannot fully rely on translators to be 100% fool proof. That's why it would be good to have a failsafe option.

Reply:
I have to agree with @Truba on this one. I would avoid doing this as well. I totally get you on "we cannot fully rely on translators", but that should be their responsibility, and if they mess something up, we should tell them to fix the translation.

It's easy when you have language such as English or German, but I wouldn't be so sure to use that on Cyrillic, Arabic, Chinese,... 😁

Reply:
Ok, I'll close this PR then